Cargo.toml

```diff
@@ -139,6 +139,7 @@ explicit_iter_loop = "warn"
 flat_map_option = "warn"
 implicit_clone = "warn"
 needless_for_each = "warn"
+needless_lifetimes = "warn"
```

Comment:
remove this as the lint should be enabled by default.

I suspect that some lint pattern was relaxed/disabled due to bug.